watch-and-repack.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+# Relative paths to the web repositories
+PENUMBRA_ZONE_WEB_PATH="../../web"
+PRAX_REPO_PATH="."
+
+# Ensure 'pack-public.sh' is executable
+chmod +x "$PENUMBRA_ZONE_WEB_PATH/pack-public.sh"
+
+# Remove any existing tarballs from packages in penumbra-zone/web 
+cleanup_tarballs() {
+  find "$PENUMBRA_ZONE_WEB_PATH/packages" -name "*.tgz" -type f -delete
```

Comment:
this is unnecessary and dangerous.

the `pack` command will easily clobber any existing tgz.

if the var is empty you will be deleting from root `/packages`. this find may also unexpectedly remove other tgz below its target, such as in `node_modules` (which symlinks out of this tree).

generally avoid deleting anything in shell scripts